crates/router/src/connector/adyen/transformers.rs

```diff
@@ -929,20 +941,30 @@ impl<'a> TryFrom<&api_models::payments::BankDebitData> for AdyenPaymentMethod<'a
                 account_number,
                 routing_number,
                 billing_details: _,
-                bank_account_holder_name,
+                card_holder_name,
+                bank_account_holder_name: _,
             } => Ok(AdyenPaymentMethod::AchDirectDebit(Box::new(
                 AchDirectDebitData {
                     payment_type: PaymentType::AchDirectDebit,
                     bank_account_number: account_number.clone(),
                     bank_location_id: routing_number.clone(),
+                    owner_name: card_holder_name.clone(),
+                },
+            ))),
+            payments::BankDebitData::SepaBankDebit {
+                billing_details: _,
+                iban,
+                bank_account_holder_name,
+            } => Ok(AdyenPaymentMethod::SepaDirectDebit(Box::new(
```

Comment:
```suggestion
            payments::BankDebitData::SepaBankDebit {
                iban,
                bank_account_holder_name,
                ..
            } => Ok(AdyenPaymentMethod::SepaDirectDebit(Box::new(
```

Same here, optional change again.